app/models/host_facet_base.rb

```diff
@@ -0,0 +1,52 @@
+class HostFacetBase < ActiveRecord::Base
+  self.abstract_class = true
+
+  def self.inherited(subclass)
```

Comment:
Shouldn't `belongs_to_host` without the wrapping method work? It's the way it works on Taxonomy